src/Apps/Collect/Routes/Collection/Components/Header/__tests__/__snapshots__/DefaultHeader.test.tsx.snap

```diff
@@ -1,87 +1,180 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`default header component renders a snapshot 1`] = `
-<header>
+<div
```

Comment:
Unless you feel strongly we should delete this snapshot in favor of assertions against actual content. Across Eigen and web there's been movement towards avoiding snapshot tests when possible.

Reply:
Ah no yeah I never use them personally, just was updating what's here. 👍